.gitignore

```diff
@@ -23,4 +23,5 @@ rls/**
 
 # ignore fuzzing corpus
 fuzz/artifacts/**
-fuzz/corpus/**
\ No newline at end of file
+fuzz/corpus/**
+*~
```

Comment:
just double checking, this is ignoring all files ending in `~`, right?

Reply:
That’s correct.